### PR TITLE
rating product

### PR DESCRIPTION
**Pull request by VuThiHien94**

### CODE REVIEW

File: app/models/rate.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class Rate < ActiveRecord::Base
-  belongs_to :user
-  belongs_to :product
+  belongs_to :rater, class_name: "User"
```
Comment (qtsoul95) [inline app/models/rate.rb]: class_name: User.name

File: app/views/shared/_show_product.html.erb [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-        <%= link_to image_tag(product.image.url),
-          product_path(product) if product.image? %>
-      </div>
-      <div class="product-name">
-        <%= link_to product.name, product_path(product) %>
-      </div>
-      <%= product.category.name %>
-      <div class="rate">
-        <%= t :rating %>
+<div class="col-md-6" data-no-turbolink="true" >
```
Comment (qtsoul95) [inline app/views/shared/_show_product.html.erb]: indent

File: app/views/shared/_show_product.html.erb [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
+      <div id="rating">
         <%= product.rating %>
       </div>
-      <%= t :price %>
-      <%= product.price_tag %>
-      <%= t :quantity %>
-      <%= product.quantity %>
-      <%= social_share_button_tag product.name,
-        url: "http://fdf_6.com/products/#{product.id}" %>
+      <div id="rate-form" >
```
Comment (qtsoul95) [inline app/views/shared/_show_product.html.erb]: indent

**Comment by duccm94** [comment]
OK

**Comment by qtsoul95** [comment]
Ok

**Pull request merged**